models/bookDao.js

```diff
@@ -0,0 +1,52 @@
+const { dataSource } = require('./dataSource');
+
+const getBookById = async (bookId) => {
+  try {
+    return await dataSource.query(
+      `
+            SELECT 
+                title, 
+                subtitle, 
+                author,
+                issue_date,
+                description,
+                thumbnail, 
+                price, 
+                is_subscribe
+            FROM books
+            WHERE books.id = ?
+            `,
+      [bookId]
+    );
+  } catch (error) {
+    error = new Error('DATABASE_CONNECTION_ERROR');
+    error.statusCode = 400;
+    throw error;
+  }
+};
```

Comment:
book 1개의 정보를 주는 것이기 때문에 배열이 아닌 객체로 반환해 주세요.
```suggestion
  try {
    const [book] = await dataSource.query(
      `
            SELECT 
                title, 
                subtitle, 
                author,
                issue_date,
                description,
                thumbnail, 
                price, 
                is_subscribe
            FROM books
            WHERE books.id = ?
            `,
      [bookId]
    );
    
    return book
  } catch (error) {
    error = new Error('DATABASE_CONNECTION_ERROR');
    error.statusCode = 400;
    throw error;
  }
};
```